.github/workflows/changelog.yml

```diff
@@ -110,6 +112,14 @@ jobs:
         env:
           BASE_BRANCH: ${{ github.base_ref }}
 
+          - name: documentation changelog check
+        if: steps.filter.outputs.documentation == 'true'
+        run: |
+          git fetch --no-tags origin main
+          towncrier check --compare-with origin/main --dir docs
+        env:
+          BASE_BRANCH: ${{ github.base_ref }}
```

Comment:
Please revert.

Reply:
so, where to put then the changelog for docs if not under `/docs/news`?

Reply:
`packages/volto/news`

Reply:
Done, moved and cleaned the mess up into one commit.